aperturedb/TensorFlowData.py

```diff
@@ -0,0 +1,45 @@
+from typing import List, Tuple
+from aperturedb.Subscriptable import Subscriptable
+import tensorflow as tf
+
+
+class TensorFlowData(Subscriptable):
+    """
+    **Class to wrap around a Dataset retrieved from [Tensorflow datasets](https://www.tensorflow.org/datasets)**
+
+    The dataset in this case can be iterated over.
+    So the only thing that needs to be implemented is __init__ and generate_query,
+    which takes an index and returns a query.
+
+    :::note
+    This class should be subclassed with a specific (custom) implementation of generate_query(),
+    and __init__ should be called with the dataset to be wrapped.
+    :::
+
+    Example subclass: [Cifar10DataTensorflow](https://github.com/aperture-data/aperturedb-python/blob/develop/examples/Cifar10DataTensorflow.py)
+
+    """
+
+    def __init__(self, dataset: tf.data.Dataset) -> None:
+        self.loaded_dataset = [t for t in dataset]
```

Comment:
Do we necessarily want to listify an arbitrarily large dataset here?

Reply:
yes, this can be removed. will do so.